benchmarks/benchmarks/analysis/rms.py

```diff
@@ -33,10 +34,13 @@ def setup(self, num_atoms, weights, center, superposition):
             self.A = self.u.atoms.positions.copy()[:num_atoms]
             self.u.trajectory[-1]
             self.B = self.u.atoms.positions.copy()[:num_atoms]
+            self.weights = [self.A.masses() / np.sum(self.A.masses()),self.B.masses() / np.sum(self.B.masses())] if 'weights' in self.params[1] else None
```

Comment:
```suggestion
            self.weights = [self.A.masses() / np.sum(self.A.masses()),self.B.masses() / np.sum(self.B.masses())] if weights else None
```
I feel the `weights` parameter should now be renamed to `use_weights`, for clarity.